patches/device/sony/lena/0002-hybris-camxoverridesettings-Disable-tele-and-uwide-l.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Bj=C3=B6rn=20Bidar?= <bjorn.bidar@jolla.com>
+Date: Mon, 28 Mar 2022 06:39:33 +0300
+Subject: [PATCH] (hybris) camxoverridesettings: Disable tele and uwide lenses
+ for now
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+MultiCamera support sadly creates issues when switching between
+Camera's in Camera1 API mode disable it to avoid those.
+After we have Camera2 API support we can revert this.
+
+This will disable tele and uwide lenses.
+
+Re:
+https://github.com/sonyxperiadev/bug_tracker/issues/732
```

Comment:
Support Multiple rear camera 
```suggestion
Support for multiple rear camera lenses sadly creates issues when
switching between cameras in Camera1 API mode. Disable it to
avoid those. After we have Camera2 API support we can revert this.

This will disable sensors of tele and uwide lenses.

Re: https://github.com/sonyxperiadev/bug_tracker/issues/732
```